src/Sarif.Multitool/ValidateCommand.cs

```diff
@@ -46,7 +46,7 @@ protected override SarifValidationContext CreateContext(ValidateOptions options,
         }
 
         private static string s_CurrentSchemaLocation = 
-            Path.Combine(Path.GetDirectoryName(typeof(ValidateCommand).Assembly.Location), "sarif-schema.json");
+            Path.Combine(Path.GetDirectoryName(typeof(ValidateCommand).Assembly.Location), "sarif-schema-2.1.0-rtm.0.json");
```

Comment:
>- [](start = 104, length = 1)

Here as well `VersionConstants.SchemaVersionAsPublishedToSchemaStoreOrg`.

Reply:
And in the project files I can just use the property from build.props! Hold on for another iteration.

---
In reply to: [285310953](https://github.com/microsoft/sarif-sdk/pull/1493#discussion_r285310953) [](ancestors = 285310953)